issues/1778/src/entity/dest.rs

```diff
@@ -0,0 +1,31 @@
+//! SeaORM Entity. Generated by sea-orm-codegen 0.9.1
```

Comment:
My bad, yep—I originally did it with 0.11 and regenerated for this example. Will try it with 0.12.

Reply:
Reran with the latest version on master, which was `0.12.rc-4`. Couldn't find a branch or anything for `rc-5`.

I also moved the files around into a separate module per schema (had mistakenly overwritten one mod/prelude with the other so the error looked the same), since that's the way my actual entities are organized, but I'm realizing I don't think that's standardized by sea-orm in any way I know of. Not sure if I got the idea of separating the schemata like that from an example of y'all's or if I just made that up.

If that is the intended organization (`entity::schema::table::model`), then I was thinking it should just have gone one level up (like `super::super::src::src::id`) but if it's not then that won't work in general. I guess the only like real danger with the current behavior is if you had a table with the same name in each schema, then the foreign key wouldn't give the compile error since it would resolve, just to the wrong table.

Reply:
https://crates.io/crates/sea-orm/0.12.0-rc.5

Reply:
Done!